.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+---
+
+_Please add `bug` and one or more of the following labels to your issue:_
+`frontend backend community dependencies`
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**OpenKAT version**
+Should contain at least the commit or release of Rocky. If you think another module may be the cause of the bug, note that module's commit or release here as well.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser: [e.g. chrome, safari]
+ - Version: [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser: [e.g. stock browser, safari]
+ - Version: [e.g. 22]
```

Comment:
This is currently phrased in a way that always ask this information, but in a lot of cases this isn't relevant. I would see browser/OS as additional context / info that only needs to be provided when relevant.